blueprints/woo-shipping/blueprint.json

```
@@ -0,0 +1,52 @@
+{
+	"$schema": "https://playground.wordpress.net/blueprint-schema.json",
+	"meta": {
+		"title": "WooCommerce Custom Flat Rate Shipping Demo",
```

There is more value to this blueprint than a _Flat Rate Shipping Demo_.
It's a really good way to test WooCommerce with a minimal configuration that allows you to use the checkout flow.

What if the name and description reflected that? The name could be something like this.

```suggestion
		"title": "WooCommerce with shipping and payment methods",
```